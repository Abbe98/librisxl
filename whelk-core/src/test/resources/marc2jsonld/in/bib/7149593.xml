<?xml version="1.0" encoding="UTF-8"?>
<record xmlns="http://www.loc.gov/MARC21/slim" type="Bibliographic">
  <leader>00887cam a2200277 a 4500</leader>
  <controlfield tag="001">7149593</controlfield>
  <controlfield tag="005">20040809152032.0</controlfield>
  <controlfield tag="008">960214s1996    sw            000 0aswe  </controlfield>
  <datafield tag="020" ind1=" " ind2=" ">
    <subfield code="a">91-0-056322-6 (inb.) ;</subfield>
    <subfield code="c">310:00</subfield>
  </datafield>
  <datafield tag="035" ind1=" " ind2=" ">
    <subfield code="9">9100563226</subfield>
  </datafield>
  <datafield tag="040" ind1=" " ind2=" ">
    <subfield code="a">NB</subfield>
  </datafield>
  <datafield tag="042" ind1=" " ind2=" ">
    <subfield code="9">BULB</subfield>
    <subfield code="9">NB</subfield>
    <subfield code="9">SEE</subfield>
    <subfield code="9">SLB</subfield>
    <subfield code="9">KVIN</subfield>
  </datafield>
  <datafield tag="084" ind1=" " ind2=" ">
    <subfield code="a">Gcdz Jansson, Tove</subfield>
    <subfield code="2">kssb/6</subfield>
  </datafield>
  <datafield tag="084" ind1=" " ind2=" ">
    <subfield code="a">Ibz Pietilä, Tuulikki</subfield>
    <subfield code="2">kssb/6</subfield>
  </datafield>
  <datafield tag="084" ind1=" " ind2=" ">
    <subfield code="a">Lz</subfield>
  </datafield>
  <datafield tag="100" ind1="1" ind2=" ">
    <subfield code="a">Jansson, Tove,</subfield>
    <subfield code="d">1914-2001</subfield>
  </datafield>
  <datafield tag="245" ind1="0" ind2="0">
    <subfield code="a">Anteckningar från en ö /</subfield>
    <subfield code="c">Tove Jansson, Tuulikki Pietilä</subfield>
  </datafield>
  <datafield tag="260" ind1=" " ind2=" ">
    <subfield code="a">Stockholm :</subfield>
    <subfield code="b">Bonnier,</subfield>
    <subfield code="c">1996 ;</subfield>
    <subfield code="e">(Finland)</subfield>
  </datafield>
  <datafield tag="300" ind1=" " ind2=" ">
    <subfield code="a">102, [1] s. :</subfield>
    <subfield code="b">ill. ;</subfield>
    <subfield code="c">25 cm</subfield>
  </datafield>
  <datafield tag="500" ind1=" " ind2=" ">
    <subfield code="a">Även tillg. med tryckår: 2. uppl., 1996</subfield>
  </datafield>
  <datafield tag="599" ind1=" " ind2=" ">
    <subfield code="a">Li:S</subfield>
  </datafield>
  <datafield tag="600" ind1="1" ind2="4">
    <subfield code="a">Jansson, Tove,</subfield>
    <subfield code="d">1914-2001</subfield>
  </datafield>
  <datafield tag="600" ind1="1" ind2="4">
    <subfield code="a">Pietilä, Tuulikki</subfield>
  </datafield>
  <datafield tag="700" ind1="1" ind2=" ">
    <subfield code="a">Pietilä, Tuulikki,</subfield>
    <subfield code="d">1917-</subfield>
  </datafield>
  <datafield tag="976" ind1=" " ind2="2">
    <subfield code="a">Gcdz</subfield>
    <subfield code="b">Litteraturhistoria Finlandssvensk</subfield>
  </datafield>
  <datafield tag="976" ind1=" " ind2="2">
    <subfield code="a">Ibz</subfield>
    <subfield code="b">Konst Konsthistoria</subfield>
  </datafield>
</record>
