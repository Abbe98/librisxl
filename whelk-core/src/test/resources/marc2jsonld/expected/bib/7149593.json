{
  "@context": "/context/libris.jsonld",
  "@id": "http://libris.kb.se/bib/7149593/data",
  "@type": "Document",
  "controlNumber": "7149593",
  "creator": {"@type": "Organization", "abbr": "NB"},
  "modified": "2004-08-09T15:20:32.0",
  "catForm": "AACR2/KRS (normalv\u00e4rde)",
  "encLevel": {
    "code":"_",
    "label": "Nationalbibliografisk niv\u00e5 (jfr KRS 1.0D3)"
  },
  "localSystemNumber": "9100563226",
  "status": {"code": "c"},
  "marc:bibliographyCode": ["SEE", "BULB", "SLB", "NB", "KVIN"],
  "about": {
    "@id": "http://libris.kb.se/bib/7149593",
    "@type": ["Instance", "Book"],
    "marc:typeOfRecord": "Text: tryck",
    "baseMaterial": "ill.",
    "comment": "\u00c4ven tillg. med tryck\u00e5r: 2. uppl., 1996",
    "dimensions": "25 cm",
    "extent": "102, [1] s.",
    "identifier": "91-0-056322-6 (inb.)",
    "isbn": "91-0-056322-6",
    "isbnNote": "inb.",
    "placeOfManufacture": {"@type": "Place", "label": "Finland"},
    "placeOfPublication": {"@type": "Place", "label": "Stockholm"},
    "publisher": {"@type": "Organization", "name": "Bonnier"},
    "pubDate": {"@type": "year", "@value": "1996"},
    "termsOfAvailability": "310:00",
    "instanceOf": {
      "@type": ["Work", "Text"],
      "marc:bibLevel": "Monografisk resurs",
      "title": "Anteckningar fr\u00e5n en \u00f6",
      "statementOfResponsibility": "Tove Jansson, Tuulikki Pietil\u00e4",
      "authorList": [
        {
          "@type": "Person",
          "authorizedAccessPoint": "Jansson, Tove, 1914-2001",
          "birthYear": "1914",
          "deathYear": "2001",
          "familyName": "Jansson",
          "givenName": "Tove",
          "name": "Tove Jansson",
          "authoritativeName": "Jansson, Tove"
        },
        {
          "@type": "Person",
          "authorizedAccessPoint": "Pietil\u00e4, Tuulikki, 1917-",
          "birthYear": "1917",
          "familyName": "Pietil\u00e4",
          "givenName": "Tuulikki",
          "name": "Tuulikki Pietil\u00e4",
          "authoritativeName": "Pietil\u00e4, Tuulikki"
        }
      ],
      "language": "swe",
      "subject": [
        {"@id": "http://dbpedia.org/resource/Fiction"},
        {"@id": "http://libris.kb.se/sab/Ibz/Pietil\u00e4,+Tuulikki+1917-"},
        {"@id": "http://dbpedia.org/resource/Autobiography"},
        {"@id": "http://libris.kb.se/sab/Gcdz/Jansson,+Tove+1914-2001"}
      ]
    }
  },
  "marc21": {
      "fields": [
        {
            "084": {
                "ind1": " ",
                "ind2": " ",
                "subfields": [
                    {
                        "a": "Gcdz Jansson, Tove"
                    },
                    {
                        "2": "kssb/6"
                    }
                ]
            }
        },
        {
            "084": {
                "ind1": " ",
                "ind2": " ",
                "subfields": [
                    {
                        "a": "Ibz Pietil√§, Tuulikki"
                    },
                    {
                        "2": "kssb/6"
                    }
                ]
            }
        },
        {
            "084": {
                "ind1": " ",
                "ind2": " ",
                "subfields": [
                    {
                        "a": "Lz"
                    }
                ]
            }
        },
        {
            "599": {
                "ind1": " ",
                "ind2": " ",
                "subfields": [
                    {
                        "a": "Li:S"
                    }
                ]
            }
        }
      ]
  }
}
